test/modules/load-gen/Settings.cs

```diff
@@ -42,6 +44,8 @@ class Settings
             this.ModuleId = Preconditions.CheckNonWhiteSpace(moduleId, nameof(moduleId));
             this.SenderType = senderType;
             this.Priorities = priorities;
+            this.Ttls = ttls;
```

Comment:
do we need to validate that ttls and priorities and ttlThresholdSecs should provide all or nothing? and if provided, size of ttls and priorites are the same (not allow empty list).

Reply:
Did some extra validation